cpu/lpc2387/lpc23xx-iap.c

```diff
@@ -38,7 +38,7 @@ static unsigned int iap_command[5];
 static unsigned int iap_result[2];
 /* typedefinition for IAP entry function */
 typedef void (*IAP)(unsigned int[], unsigned int[]);
-IAP IAP_Entry;
+static const IAP IAP_Entry = (IAP)0x7ffffff1;
```

Comment:
This variable seemed to have been unused previously, but now is used correctly :+1: